app/src/main/java/org/schabi/newpipe/error/ReCaptchaActivity.java

```diff
@@ -107,12 +106,7 @@ public void onPageFinished(final WebView view, final String url) {
         // cleaning cache, history and cookies from webView
         recaptchaBinding.reCaptchaWebView.clearCache(true);
         recaptchaBinding.reCaptchaWebView.clearHistory();
-        final CookieManager cookieManager = CookieManager.getInstance();
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            cookieManager.removeAllCookies(value -> { });
-        } else {
-            cookieManager.removeAllCookie();
-        }
+        CookieManager.getInstance().removeSessionCookies(null);
```

Comment:
Why you now remove only session cookies ([i.e.](https://developer.android.com/reference/android/webkit/CookieManager#removeSessionCookies(android.webkit.ValueCallback%3Cjava.lang.Boolean%3E)) *cookies without an expiration date*), instead of all cookies with [`removeAllCookies(null)`](https://developer.android.com/reference/android/webkit/CookieManager#removeAllCookies(android.webkit.ValueCallback%3Cjava.lang.Boolean%3E))?

Reply:
Looks like I selected the wrong method in the autocomplete 😅